src/FSharp.AWS.DynamoDB/Expression/ConditionalExpr.fs

```diff
@@ -579,18 +533,27 @@ type ConditionalExpression with
         | None -> None
 
     member cond.GetDebugData() =
-        let aw = new AttributeWriter()
+        let aw = new AttributeWriter ()
         let expr = writeConditionExpression aw cond
-        let names = aw.Names |> Seq.map (fun kv -> kv.Key, kv.Value) |> Seq.toList
-        let values = aw.Values |> Seq.map (fun kv -> kv.Key, kv.Value.Print()) |> Seq.toList
+
+        let names =
+            aw.Names
+            |> Seq.map (fun kv -> kv.Key, kv.Value)
```

Comment:
maaaan (though the answer is to rewrite it as `[ for kv in aw.Names -> kv.Key, kv.Value]`)